### PR TITLE
Fix tani freq

### PR DESCRIPTION
**Pull request by kfir4444**
Fixes the number of frequencies returned from the torchani adapter, based on the [torchani user guide](https://aiqm.github.io/torchani/examples/vibration_analysis.html)

### CODE REVIEW

File: arc/job/adapters/scripts/tani_script.py [hunk 1]
```diff
@@ -154,7 +154,6 @@ def run_vibrational_analysis(xyz: dict = None,
     hessian = torchani.utils.hessian(coordinates, energies=energies)
     freqs, modes, force_constants, reduced_masses = torchani.utils.vibrational_analysis(masses, hessian, mode_type='MDU')
     freqs = freqs.numpy()
-    freqs = freqs[~np.isnan(freqs)]
```
Comment (alongd) [inline arc/job/adapters/scripts/tani_script.py:157]: Looks like np isn't used anymore, can you removed the import?

File: arc/job/adapters/torch_ani_test.py [hunk 2]
```diff
@@ -162,7 +162,6 @@ def test_run_vibrational_analysis(self):
         results = read_yaml_file(path=self.job_10.local_path_to_output_file)
         self.assertEqual(list(results.keys()), ['force_constants', 'freqs', 'hessian', 'modes', 'reduced_masses'])
         self.assertEqual(len(results['force_constants']), 3*self.job_10.species[0].mol.get_num_atoms())
-        self.assertEqual(len(results['freqs']), 3*self.job_10.species[0].mol.get_num_atoms()-5)
```
Comment (alongd) [inline arc/job/adapters/torch_ani_test.py:165]: does it pass if you leave it? If so, no harm in keeping it

Comment (kfir4444) [reply arc/job/adapters/torch_ani_test.py:165]: It doesn't pass, since torchani does not return the correct number of freqs.

**Review by Laxzal — APPROVED** [review]
looks good to me

File: arc/job/adapters/torch_ani_test.py (hunk 2 above)
Comment (alongd) [reply arc/job/adapters/torch_ani_test.py:165]: Can you correct the test instead of deleting it?

**Review by alongd — APPROVED** [review]
Thanks!

**Pull request merged**